src/main/java/enhacedCalculator/EnhancedCalculator.java

```diff
@@ -0,0 +1,24 @@
+package enhacedCalculator;
+
+public class EnhancedCalculator {
+
+    public int sum(int a, int b){
+        return a + b;
+    }
+
+    public int subtract(int a, int b) {
+        return a - b;
+    }
+
+    public int multiply(int a, int b) {
+        return a * b;
+    }
+
+    public double divide(int a, int b) {
+            return (a / (double) b);
```

Comment:
to działa, ale za dużo zalet obiektowości to my tutaj nie wykorzystujemy ;) 

![image](https://user-images.githubusercontent.com/17989854/125898696-2bd615ee-7004-4913-a585-a31092d732e6.png)

nie chodzi tutaj o dodanie kolejnej metody w tym miejscu. Wyobraź sobie, że chcemy teraz dodać 10 operacji z których każda jest bardziej skomplikowana niż jedna linijka (np. 300 linijek). W pewnym momencie ten plik będzie mial pare tysięcy linii i stanie się nieutrzymywalny.

Podpowiedź - sprawdź jaka jest część wspólna tych metod, użyj interfejsów. :)